templates/terraform/examples/firebase_web_app_basic.tf.erb

```diff
@@ -0,0 +1,41 @@
+resource "google_project" "default" {
+	provider = google-beta
+
+	project_id = "tf-test%{random_suffix}"
+	name       = "tf-test%{random_suffix}"
+	org_id     = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "google_firebase_project" "default" {
+	provider = google-beta
+	project  = google_project.default.project_id
+}
+
+resource "google_firebase_web_app" "<%= ctx[:primary_resource_id] %>" {
+	provider = google-beta
+	project = google_project.default.project_id
+	display_name = "<%= ctx[:vars]['display_name'] %>"
+
+	depends_on = [google_firebase_project.default]
+}
+
+resource "google_storage_bucket" "default" {
+    provider = google-beta
+    name = "<%= ctx[:vars]['bucket_name'] %>"
+}
+
+resource "google_storage_bucket_object" "default" {
```

Comment:
This doesn't appear to be testing anything so I'm assuming it's mostly here for the generated documentaiton.
Can you add a comment here what the purpose of this bucket object is? It will help readers, such as myself, understand what the firebase-config is for. Bonus points for linking to documentation.